src/main/java/org/gridsuite/cases/importer/job/SftpCaseAcquisitionJob.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.cases.importer.job;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Date;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * @author Nicolas Noir <nicolas.noir at rte-france.com>
+ */
+public class SftpCaseAcquisitionJob {
+
+    public static void main(String... args)
+            throws IOException, InterruptedException {
+
+        // Loading properties
+        Properties cassandraProperties = new Properties();
```

Comment:
Why did you choose to rely on properties for config? Why several? Why not yaml? Why not using powsybl platform config?

Reply:
Since the case import job is planned to be used as a simple java client, it might not be necessary to use a specific configuration format like yaml. Properties reading is easy to achieve with Java.

Reply:
It might not be necessary to add a dependency with the Powsybl platform config, since the job is a simple jar.

Reply:
You just need to add a dependency to powsybl-classic-config. No need to deal with json directly, PlatformConfig has a Java simple API to read property like data backed by either a yaml, an xml or properties (deprecated).
Java properties are really old fashion.

Reply:
Configuration of the job is now defined in a single configuration file (a yaml) and read using Powsybl platform config module.